admin/proc-running-rhdh-behind-a-proxy.adoc

```diff
@@ -0,0 +1,94 @@
+[id='proc-running-rhdh-behind-a-proxy_{context}']
+= Running the {product-very-short} application behind a corporate proxy
+
+You can run the {product-very-short} application behind a corporate proxy by setting any of the following environment variables before starting the application:
+
+* `HTTP_PROXY`: Denotes the proxy to use for HTTP requests.
+* `HTTPS_PROXY`: Denotes the proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The variable value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
+.Prerequisites
+
+* You have installed the {product}. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+* You have created a project in the {product-short}.
+
+.Procedure
+
+. Configure proxy information in one of the following deployments based on your needs:
++
+.Helm deployment
+
+* Set the proxy information in your Helm configuration file:
++
+[source,yaml]
+----
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        value: '<http_proxy_url>'
+      - name: HTTPS_PROXY
+        value: '<https_proxy_url>'
+      - name: NO_PROXY
+        value: '<no_proxy_settings>'
+----
++
+Where,
+
+`<http_proxy_url>`:: Denotes a variable that you must replace with the HTTP proxy URL.
+`<https_proxy_url>`:: Denotes a variable that you must replace with the HTTPS proxy URL.
+`<no_proxy_settings>`:: Denotes a variable that you must replace with comma-separated URLs, which you want to exclude from proxying, for example, `foo.com,baz.com`.
++
+.Example: Setting proxy variables using Helm Chart
+
+[source,yaml]
+----
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        value: 'http://10.10.10.105:3128'
+      - name: HTTPS_PROXY
+        value: 'http://10.10.10.106:3128'
+      - name: NO_PROXY
+        value: 'localhost,example.org'
+----
+
++
+.Operator deployment 
+
+** Perform one of the following steps:
+* Set the proxy information in a ConfigMap file that contains default configuration:
++
+.. Search for a ConfigMap file named `backstage-default-config` in the namespace where the Operator is deployed and open it. 
+.. Find the `deployment.yaml` key.
+.. Add the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables to the containers listed in the `Deployment` spec as shown in the following example:
```

Comment:
@rm3l, I think we should not put this piece of information in a note because this is also a key step for configuring proxy information. Here, the user can make a decision whether they want to go with proxy configuration in a ConfigMap file or an individual CR file based on their requirements.

So, I have made this step part of the procedure and notified users to perform one of these two steps.

Reply:
That makes sense. But maybe it is worth calling out that editing the operator default config is the responsibility of an administrator (with access to the operator namespace). So it can be a way for a cluster admin to apply the proxy settings to all the operator users.
The other alternative, at the individual CR level, can be done by regular users.

Reply:
Which role would configure Helm configuration?? adminstrators, developers, or both of them.

Reply:
For Helm, it is quite different. Each Helm-based instance is deployed from a user-provided `values` file. So a regular user would need to add the proxy settings to their `values` file(s).

Reply:
I included all these details in the into para because providing these deatils in the middle of the procedure distracts the user from the procedure.